SCClassLibrary/Common/GUI/Base/enums.sc

```diff
@@ -253,3 +253,13 @@ QToolButtonStyle {
 
 	*new { arg name; ^this.perform(name) }
 }
+
+QScrollHint {
+	classvar
```

Comment:
these could be `const`.

Reply:
All the enum values in enums.sc should be const - but, in general, the enums stuff needs a little clean-up and factoring. Mind if I address separately? I'd rather match style and implementation with this checkin, and deal with enum issues all at once.